packages/scandipwa/src/component/ProductBundleItems/ProductBundleItems.container.js

```diff
@@ -117,26 +174,43 @@ export class ProductBundleItemsContainer extends ProductCustomizableOptionsConta
             selectedCheckboxValues = []
         } = this.state;
 
+        const { isDynamicPrice } = this.props;
+
         const values = [...selectedCheckboxValues, ...selectedDropdownOptions];
 
         if (values.length) {
-            return this.getOptionPrice(item, values);
+            return this.getOptionPrice(item, values, isDynamicPrice);
         }
 
-        return { price: 0, priceExclTax: 0 };
+        return { price: 0, priceExclTax: 0, initialPrice: 0 };
     };
 
     getTotalPrice() {
-        const { items } = this.props;
+        const { items, isDynamicPrice } = this.props;
+
+        const {
+            price_range: {
+                minimum_price: {
+                    default_price: { value: defaultPrice } = {},
+                    default_final_price: { value: defaultFinalPrice } = {},
+                    default_final_price_excl_tax: { value: defaultFinalPriceExclTax } = {}
+                }
+            }
+        } = this.props;
+
+        const price = isDynamicPrice ? 0 : defaultPrice;
+        const finalPrice = isDynamicPrice ? 0 : defaultFinalPrice;
+        const priceExclTax = isDynamicPrice ? 0 : defaultFinalPriceExclTax;
 
         return items
-            .map(this.getItemsPrice)
+            .map(this.getItemsPrice.bind(this))
```

Comment:
Knowing that `getItemsPrice` is declared as an arrow function, binding it to anything should not produce any result - it is bound to the instance automatically, and rebinding stuff is not allowed by JS.